packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -119,6 +133,100 @@ export function ModelsTree(props: ModelsTreeProps) {
   );
 }
 
+interface ModelsTreeRendererProps extends TreeRendererBaseProps {
+  nodeRendererProps: ModelsTreeNodeRendererProps;
+}
+
+function createModelsTreeRenderer(
```

Comment:
Remove `createModelsTreeRenderer` and move customization to `ModelsTreeRendererProps`.